tests/concepts/twrapconcept.nim

```diff
@@ -1,7 +1,6 @@
 discard """
   errormsg: "type mismatch: got <string>"
-  line: 21
-  nimout: "twrapconcept.nim(11, 5) Foo: concept predicate failed"
+  nimout: "twrapconcept.nim(10, 5) Foo: concept predicate failed"
```

Comment:
actually `line` is silently ignored when errormsg is not specified but is honored when errormsg is specified.

That being said, `errormsg` is redundant with nimout since nimout can specify the error line. Right now you've weaked the test by removing the line number where the error occurs, but the following should be used instead: 

it contains all the original information (line nnumber and error msg), is not ambiguous, and also contains column, and is easier to maintain / more systematic.

Other tests could use this too (at least ones in this PR where you've removed line number AND an errormsg was specified)

```
discard """
  action: reject
  nimout: '''
twrapconcept.nim(23, 14) Error: type mismatch: got <string>
twrapconcept.nim(13, 5) Foo: concept predicate failed
'''
"""

# https://github.com/nim-lang/Nim/issues/5127

type
  Foo = concept foo
    foo.get is int

  FooWrap[F: Foo] = object
    foo: F

proc get(x: int): int = x

proc wrap[F: Foo](foo: F): FooWrap[F] = FooWrap[F](foo: foo)

let x = wrap(12)
let y = wrap "string"
```